.cicd/platforms/ubuntu18.Dockerfile

```diff
@@ -2,12 +2,21 @@ FROM ubuntu:bionic
 
 RUN apt-get update && apt-get upgrade -y && \
     DEBIAN_FRONTEND=noninteractive apt-get install -y build-essential \
-                                                      cmake \
                                                       g++-8 \
                                                       curl \
                                                       ninja-build \
                                                       software-properties-common \
-                                                      zlib1g-dev
+                                                      zlib1g-dev \
+                                                      pkg-config \
+                                                      libboost-all-dev \
+                                                      libcurl4-gnutls-dev
+
+RUN curl -L https://cmake.org/files/v3.13/cmake-3.13.5.tar.gz | tar zx && \
```

Comment:
If we were planning to support Ubuntu 18 ongoing I would suggest we use version 3.16.3 or 3.22.1 to match U 20 or 22 respectively. This is just a comment, no action needs to be taken here.

Reply:
since we override default `apt-get` version your point makes sense but I do not think it worth experimenting with that at the moment